.github/workflows/checks.yml

```diff
@@ -0,0 +1,23 @@
+name: "Checks"
+on: [pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Yarn PnP Setup
```

Comment:
Is it needed? We did not run it previously

Reply:
it's just a boilerplate action that I use to set up yarn and cache, you can see the action here: https://github.com/Araxeus/setup-yarn-pnp-action/blob/main/action.yml

It isn't really needed, I could revert this change if you want

Reply:
Sorry, I'm fine with the setup action, I just meant: do we need to do the setup for the dependency review? We only did the checkout previously

Reply:
Ooh wait, my bad… you're right!